src/views/PrintStatus.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {
+    AppBar,
+    Button,
+    Grid,
+    Paper,
+    Typography
+} from "@material-ui/core";
+import { ThemeProvider } from '@material-ui/core/styles';
+import DoneOutlineIcon from '@material-ui/icons/DoneOutline';
+import HourglassEmptyIcon from '@material-ui/icons/HourglassEmpty';
+
+import theme from "../theme";
+
+class PrintStatus extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      uploading: false,
+      printing: false,
+      filesToBePrinted: 0
+    }
+  }
+
+  componentDidMount() {
+    //GET NUMBER OF FILES TO BE PRINTED?
+  }
+
+  render() {
+    return (
+            <div>
+                <ThemeProvider theme={theme}>
+                    <Paper>
+                        <AppBar position="static" alignitems="center" color="primary" style={{ marginBottom: '20px' }}>
+                                <Grid container justify="center" wrap="wrap">
+                                    <Grid item>
+                                        <Typography variant="h4">View Printing Status</Typography>
+                                    </Grid>
+                                </Grid>
+                        </AppBar>
+                        {!this.state.uploading && !this.state.printing && <Grid container justify="center" wrap="wrap">
```

Comment:
Use destructuring like this:

`const {uploading, printing} = this.state`

Then in this line you can add: `{!uploading && !printing && <Grid...`

Reply:
Done